safe/gui/tools/wizard/step_kw55_source.py

```diff
@@ -143,9 +140,12 @@ def set_widgets(self):
         source_date = self.parent.get_existing_keyword('date')
         if source_date:
             self.ckbSource_date.setChecked(True)
-            self.dtSource_date.setDateTime(
-                QDateTime.fromString(get_unicode(source_date),
-                                     'yyyy-MM-dd HH:mm:ss'))
+            try:
+                self.dtSource_date.setDateTime(source_date)
+            except TypeError:
+                raise TypeError(
```

Comment:
Ah, thanks. Fix it. The original TypeError give the same exception message, so I will delete it.